src/aggregate/aggregate_exec.c

```diff
@@ -619,6 +619,8 @@ static void sendChunk_Resp3(AREQ *req, RedisModule_Reply *reply, size_t limit,
     } else if (rc == RS_RESULT_ERROR) {
       // Non-fatal error
       RedisModule_Reply_SimpleString(reply, QueryError_GetError(req->qiter.err));
+    } else if (req->qiter.err->reached_maxprefixexpansions) {
+      RedisModule_Reply_SimpleString(reply, "Max prefix expansions limit was reached");
```

Comment:
I suggest to clearly add the actual value of the limit (make sure the expected limit is indeed in place)

Reply:
Since we assume this value is **known**, since the user either explicitly configures it or can get it via the config parameters' values, I think it's quite redundant..

Reply:
This could reduce ping-pong with users, asking for more info.